### PR TITLE
FEATURE: Flow 7.x compatibility

### PR DESCRIPTION
**Pull request by bwaidelich**
Related: https://github.com/neos/flow-development-collection/issues/2244

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
     "description": "Google Analytics integration for Neos CMS",
     "license": "GPL-3.0-or-later",
     "require": {
-        "neos/neos": "~4.3 || ~5.0 || dev-master",
-        "neos/fusion-afx": "^1.2",
+        "neos/neos": "^7.0 || dev-master",
+        "neos/fusion-afx": "^7.0",
```
Comment (Sebobo) [inline composer.json:8]:
```suggestion
```

Comment (Sebobo) [reply composer.json:8]: Think we don't need this explicitly

**Review by Sebobo — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (kdambekalns) [reply composer.json:8]: We do, the code uses AFX… Relying on it "to be there already" is doubtful.

**Review by kdambekalns — APPROVED** [review]

**Comment by bwaidelich** [comment]
@paavo I don't understand the question? This adds support for 7+ and drops support for lower versions. It was released with  [3.3.0](https://github.com/neos/neos-googleanalytics/releases/tag/3.3.0). For 5.x support, you can install an older version (`^3.2`). Does that help?

**Comment by paavo** [comment]
I thought support for Neos 5.x was dropped and the Analytics Tag was only updated in the 3.3 branch for Neos 7+..
then i realized we used the TagManager setting instead of analytics 🤦 
Sorry for the confusion @bwaidelich

**Comment by bwaidelich** [comment]
No problem :)

**Pull request merged**